run-qa-checks

```diff
@@ -23,12 +23,12 @@ echo 'Dockerfile QA checks...'
 # DL3013 - Pin versions in pip. Instead of `pip install <package>` use `pip install <package>==<version>`
 # DL3007 - Using latest is prone to errors if the image will ever update. Pin the version explicitly to a release tag
 # DL3018 - Pin versions in apk add. Instead of `apk add <package>` use `apk add <package>=<version>`
```

Comment:
I think this step will be right direction.

We will need to follow the release of dependency packages more closely. We can leverage "Depandabot" for that. 

On a personal note, I am fed up of random failing builds.

Reply:
I agree in pinning the version and tagging build.
I have one doubt in python packages i.e. for openwisp packages we are using the github master tarball and we cannot specify version in this case, so should I ignore these lines?
https://github.com/openwisp/docker-openwisp/blob/ad61dfd26584d69de3651cd8935d2f0b73115740/images/openwisp_base/Dockerfile#L28-L36

Reply:
openwisp-radius 1.0.0 is out and the other modules will be released very soon, so we'll pin them all

Reply:
> I think this step will be right direction.
> 
> We will need to follow the release of dependency packages more closely. We can leverage "Depandabot" for that.

Yes, we should be using dependabot more, it's working pretty well in wifi-login-pages after we introduced browser testing.
 
> On a personal note, I am fed up of random failing builds.

Me too!

Reply:
The only reason I was not pinning versions is because of the fact docker-openwisp is not in beta yet.
But if we are focusing on it, now, I think it'll be good to pin them! 😄

Reply:
@atb00ker Is there any stable tag for `openwisp-base`?
I need to specify the tag in lines like `FROM openwisp/openwisp-base:latest` because using latest tag is giving warning.
I see few tags here: https://hub.docker.com/r/openwisp/openwisp-base/tags. Can you suggest which one to use except latest?

Reply:
@codesankalp 
We can ignore that warning, because latest is build just before the images are build on local, so we want it to be `latest` only! 😄